### PR TITLE
fix: replace tkn.farm

### LINKED ISSUE #2223: [258] Token farm functionality issue
**Title:** 
Token farm functionality issue
**Description:** 
Token farm no longer works 
**Summary:** 
The customer is experiencing an issue with the token farm feature, which is no longer functioning. This appears to be a critical issue and should be addressed promptly to avoid further dissatisfaction.
## Info
**Reported by:** Guest [(ska7eordie@gmail.com)](mailto:ska7eordie@gmail.com) - [Open in Gleap](https://app.gleap.io/projects/6272ba4eb62acc0036624b99/bugs/2yR21D67SqNhae6sUC2vFGu7g3pmqqt1l7YZRk3tGslTFhgwOtqBffnvtruwbDWeC7ud7N)
**Location:** US
**Type:** 🚨 BUG

🔗 [Open share link](https://app.gleap.io/share/2yR21D67SqNhae6sUC2vFGu7g3pmqqt1l7YZRk3tGslTFhgwOtqBffnvtruwbDWeC7ud7N?token=DYOuhTpNpC6Y7zGscMhQKfRniWHgavLJR1K1r5at0noQniwCDY0eB0tDRB5OO9DVgIhfaq)
🌍 [Open ticket in Gleap](https://app.gleap.io/projects/6272ba4eb62acc0036624b99/bugs/2yR21D67SqNhae6sUC2vFGu7g3pmqqt1l7YZRk3tGslTFhgwOtqBffnvtruwbDWeC7ud7N)
## Metadata
**browserName:** Safari(17.5)
**userAgent:** Mozilla/5.0 (iPhone; CPU iPhone OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.5 Mobile/15E148 Safari/604.1
**browser:** Safari
**systemName:** iPhone
**sessionDuration:** 481
**devicePixelRatio:** 3
**screenWidth:** 393
**screenHeight:** 852
**innerWidth:** 393
**innerHeight:** 741
**currentUrl:** https://docs.near.org/build/primitives/ft
**language:** en-US
**mobile:** true
**sdkVersion:** 13.7.3
**sdkType:** javascript
**environment:** prod

Screenshot

### PR DESCRIPTION
**Pull request by flmel**
Replaces tkn.farm community project with the up to date tools on [DevPortal](https://dev.near.org/tools) on Primitives/FT section

Closes #2223 

@race-of-sloths

### CODE REVIEW

**Comment by race-of-sloths** [comment]
@flmel Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
You are true leader! Other sloths hot on your heels, keep moving!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/flmel?type=bot&pr=near/docs/2259&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/flmel?type=bot&pr=near/docs/2259&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/flmel)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/flmel)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Review by garikbesson — APPROVED** [review]

**Comment by garikbesson** [comment]
That's nice to have this Toolbox on Dev Portal finally

**Comment by garikbesson** [comment]
@race-of-sloths score 1

**Review by bucanero — APPROVED** [review]

**Pull request merged**
